main.go

```diff
@@ -378,12 +378,21 @@ func main() {
 
 		os.Exit(task(token))
 	} else if cmd == "web" {
-		if opts.Web.Stop {
-			util.WebStop()
+		var directory string = "~/.oauth2l-web"
+		if len(opts.Web.Directory) > 0 {
```

Comment:
No need for the extra nested if statements. Simplify to:

if len(opts.Web.Directory) > 0 {
directory = opts.Web.Directory
}
if opts.Web.Stop {
	util.WebStop(directory)
} else {
	util.Web(directory)
}

Reply:
I was completely over complicating it, thanks for the suggestion!